lib/modules/dasboard/controller/dashboard_event.dart

```diff
@@ -0,0 +1,12 @@
+part of 'dashboard_bloc.dart';
+
+abstract class DashboardEvent {
+  const DashboardEvent();
+}
+
+class GetSelectedExamCategory extends DashboardEvent {}
+
+class GetAllExamList extends DashboardEvent {
+  String parentId;
```

Comment:
what is "parentId"

Reply:
_**id**_ defines the exam id and _**parentId**_ resembles the exam category id.